.gitignore

```diff
@@ -2,5 +2,7 @@
 *.user
 .idea/
 .ionide/
-bin/
-obj/
+/src/*/bin/
```

Comment:
Do you think it would be better to have an opt-in pattern instead for the `bin` dirs inside `lib/*`?

Reply:
By doing something fancy here? Or by sticking another gitignore inside `lib`?

Reply:
By doing something here like

```.gitignore
bin/
obj/
!/lib*/bin
```